application.properties

```diff
@@ -0,0 +1,6 @@
+
+application.info=demo sources for work with files presentation
+java.version=1.8
+
+excel.defaultName=excelFileFromLocationB
```

Comment:
Usually we don't want to keep something like excelFromA/B. Just proper working defaults.

Reply:
These names were for my own temporary use - I wanted to know if the defaults were read from correct location. Now I changed them to just "schedule", but left "scheduleJAR" for a while. If you want to make sure that it is read from JAR, just build executable JAR and run the program without application.parameters files. Your  generated schedule will get name scheduleJAR.xlsx
But I must remember to change that name when the code is finished.